src/bench/crypto_hash.cpp

```diff
@@ -60,7 +60,10 @@ static void SHA512(benchmark::State& state)
 static void FastRandom_32bit(benchmark::State& state)
 {
     FastRandomContext rng(true);
+    _Pragma("GCC diagnostic push")
+    _Pragma("GCC diagnostic ignored \"-Wunused-but-set-variable\"")
     uint32_t x = 0;
+    _Pragma("GCC diagnostic pop")
```

Comment:
```suggestion
    [[maybe_unused]] uint32_t x = 0;
```
(not sure whether this works, but if it does [it is standard C++17](https://en.cppreference.com/w/cpp/language/attributes/maybe_unused)).

Reply:
Yeah, the `_Pragma` (at least in this first pass) are just placeholders. I’ve eliminated them all now. In this case, I went with returning the value that is calculated for the benchmark. But `[[maybe_unused]]` might work here, too. Happy to try it if you think it’s a better solution. (I used `[[maybe_unused]]` in other cases, like ranged `for` loops, where you can’t omit the variable name to prove that it’s unused.)